public/client.js

```diff
@@ -59,18 +67,20 @@ socket.on('user set', function (data) {
     $("body").css("background-color","#f8f8f8");
     $(".wrapper").fadeIn();
     $(".chat[data-chat='person1']").append("<div class='conversation-start'>\
-                                                <span>" + date.getHours() + ':' + date.getMinutes() + "</span>\
-                                            </div>");
+    					        <span>" + date.getHours() + ':' + date.getMinutes() + "</span>\
```

Comment:
I don;t understand why all these are coming as change.
Have you changed tabs to spaces or spaces to tabs?

Reply:
Yes, I changed that in sublime text, but don't know how to fix. A help would be appreciated.